client/state/terms/selectors.js

```diff
@@ -26,6 +27,28 @@ export function isRequestingTermsForQuery( state, siteId, taxonomy, query ) {
 }
 
 /**
+ * Returns true if currently requesting terms for a query, excluding all known
+ * queried pages, or false otherwise.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {String}  taxonomy Taxonomy slug
+ * @param  {Object}  query    Terms query object
+ * @return {Boolean}           Terms for the query
+ */
+export function isRequestingTermsForQueryIgnoringPage( state, siteId, taxonomy, query ) {
```

Comment:
Actually this is to determine if there are any active taxonomy/term requests regardless of page for a given query - so we are looking at `state.terms.queryRequests` not the items.